common/include/pcl/pcl_macros.h

```diff
@@ -443,6 +443,8 @@ aligned_free(void* ptr)
   #define PCL_NODISCARD
 #endif
 
+#define PCL_CONSTEXPR_CTOR constexpr
```

Comment:
This isn't required. We add a macro only if there's some logic needed

Reply:
sorry, i forgot i covered all cases, so conditional constexpr is not requred anymore
fixed

Reply:
i also went through commented parts, looks like it's clean now